source/common/network/lc_trie.h

```diff
@@ -533,16 +533,19 @@ std::string LcTrie::LcTrieInternal<IpType, address_size>::getTag(const IpType& i
 
   // /0 will match all IP addresses.
   if (ip_prefixes_[address].length_ == 0) {
-    return ip_prefixes_[address].tag_;
+    return_vector.push_back(ip_prefixes_[address].tag_);
+    // TODO(ccaraman): When nested prefixes are supported, should the /0 case be handled better.
```

Comment:
super nitty nit - either swap . for ? or make it a statement please

Reply:
:) - I'll change it to a '?'.